docs/Docs.js

```diff
@@ -113,6 +117,8 @@ class Docs extends Component {
                 onBorderlessOptionsChange={this.createStateLink('borderlessItems')}
                 skipToContentHref={skipToContentHref}
                 onSkipToContentHrefChange={this.createStateLink('skipToContentHref')}
+                isRedesign={isRedesign}
+                onisRedesignChange={this.createStateLink('isRedesign')}
```

Comment:
Majorly unimportant nit...

```suggestion
                onIsRedesignChange={this.createStateLink('isRedesign')}
```